src/test/java/core/basesyntax/service/impl/CsvDataConverterTest.java

```diff
@@ -0,0 +1,123 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class CsvDataConverterTest {
+    private final List<FruitTransaction> validFruitTransactions = fillTransactionList();
+    private final DataConverter dataConverter = new CsvDataConverterImpl();
+
+    @Test
+    public void convertToTransaction_validData_Ok() {
+        List<String> actualArray = fillInputArray();
+        List<FruitTransaction> currentTransactions
+                = dataConverter.convertToTransaction(actualArray);
+        assertEquals(validFruitTransactions, currentTransactions);
+    }
+
+    @Test
+    public void convertToTransaction_invalidOperation_notOk() {
+        List<String> invalidOperationList = fillInputArray();
+        invalidOperationList.add("a,apple,20"); // wrong operation "a"
```

Comment:
you can remove this comment. Method name informative 
(the same for other methods)